test/testdata/trustroot/e2e/with-tsa.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+apiVersion: policy.sigstore.dev/v1alpha1
+kind: TrustRoot
+metadata:
+  name: my-sigstore-keys
+spec:
+  sigstoreKeys:
+    timestampAuthorities:
```

Comment:
```
  The TrustRoot "my-sigstore-keys" is invalid: spec.sigstoreKeys.certificateAuthorities: Invalid value: "null": spec.sigstoreKeys.certificateAuthorities in body must be of type array: "null"
  Error: Process completed with exit code 1.
```

Don't we still need the signing CA added? It's currently listed as 'required', so if this is a bad assumption we need to relax it in the trustroot API.

Reply:
I am still working on this ... thanks for looking at it. I am adding a WIP.

Reply:
I'll complete the tests.

Reply:
Oh no worries, was just thinking through my assumptions on how things work :) My thinking has been that there always has to be a CA, but maybe that's wrong.

Reply:
> Don't we still need the signing CA added? It's currently listed as 'required', so if this is a bad assumption we need to relax it in the trustroot API.

You can sign using the TSA as verification and a raw key without anything else.

Reply:
We should relax the validation.

Reply:
Done